### PR TITLE
[WIP] simplify CI/remove code climate

### PR DESCRIPTION
**Pull request by gaborbernat**

### CODE REVIEW

**Comment by gaborbernat** [comment]
@asottile take a look at the new format and tell me what you think

File: azure-pipelines.yml [hunk 1]
```diff
@@ -1,203 +1,73 @@
...
-    condition: succeededOrFailed()
+- template: run-tox-env.yml@tox-azure
+  parameters:
+    jobs:
+      check:
+        py: '3.7'
+        toxenvs:
+        ? fix_lint
+        ? docs
+        ? package_description
```
Comment (asottile) [inline azure-pipelines.yml]: hmmm! how does this syntax work?

Comment (gaborbernat) [reply azure-pipelines.yml]: yaml sets?

Comment (asottile) [reply azure-pipelines.yml]: wow, I've *literal*ly never seen this

Comment (asottile) [reply azure-pipelines.yml]: apparently it's a "key indicator" so it makes mappings with no values?

```pycon
>>> yaml.safe_load('''\
... x:
...   ? a
...   ? b
...   ? c
... ''')
{'x': {'a': None, 'b': None, 'c': None}}
```

Comment (gaborbernat) [reply azure-pipelines.yml]: yes per standard defintion

Comment (gaborbernat) [reply azure-pipelines.yml]: changed back to array now 👍

**Comment by gaborbernat** [comment]
I think the current design is the cleanest, and I'm proposing it as the official way to use tox in Azure Pipelines.

**Pull request merged**